features/git-ship/current_branch/on_feature_branch/without_open_changes/without_tracking_branch.feature

```diff
@@ -30,3 +30,23 @@ Feature: git ship: shipping the current feature branch without a tracking branch
     And I have the following commits
       | BRANCH | LOCATION         | MESSAGE      | FILE NAME    |
       | main   | local and remote | feature done | feature_file |
+
+
+  Scenario: undo
+    When I run `git ship --undo`
+    Then it runs the Git commands
+      | BRANCH  | COMMAND                                        |
+      | main    | git branch feature <%= sha 'feature commit' %> |
+      |         | git revert <%= sha 'feature done' %>           |
+      |         | git push origin main                           |
+      |         | git checkout feature                           |
+      | feature | git checkout main                              |
+      | main    | git checkout feature                           |
+    And I end up on the "feature" branch
+    And I have the following commits
+      | BRANCH  | LOCATION         | MESSAGE               | FILE NAME    |
+      | main    | local and remote | feature done          | feature_file |
+      |         |                  | Revert "feature done" | feature_file |
+      | feature | local            | feature commit        | feature_file |
+
+
```

Comment:
I think the editor should do this, similar to how it removes trailing whitespace on lines. The linter should also catch this. I'll look into this. Sorry about all the empty lines here.